include/vcpkg/base/messages.h

```diff
@@ -1522,6 +1571,13 @@ namespace vcpkg
         (msg::option),
         "",
         "the --{option} switch was passed, but there is no manifest file to add a `builtin-baseline` field to.");
+    DECLARE_MESSAGE(
+        UpdateBaselineHelp,
+        (),
+        "",
+        "To keep your libraries up to date, the best approach is to update your baseline reference. This will "
```

Comment:
```suggestion
        "The best approach to keep your libraries up to date is to update your baseline reference. This will "
```

I don't have a strong opinion on this one; just a style suggestion.